tests/test_cnn/test_weight_init.py

```diff
@@ -14,6 +14,9 @@
                       initialize, kaiming_init, normal_init, trunc_normal_init,
                       uniform_init, xavier_init)
 
+if torch.__version__ == 'parrots':
+    pytest.skip('not supported in parrots now', allow_module_level=True)
```

Comment:
Does `parrots` not support `weight_init`?

Reply:
Parrots supports weight_init, but Tensor.allclose can't support broadcast in parrots.So we just skip it temporarily. We will delete this skip code when parrots supports it.

Reply:
Got it.